typescript-typing-tutor/main.ts

```diff
@@ -0,0 +1,38 @@
+document.addEventListener('DOMContentLoaded', () => {
+  const spans = document.querySelectorAll('#phrase span');
+  let currentIndex = 0;
+
+  function updateCurrentSpan(): void {
+    spans.forEach((span, index): void => {
+      if (index === currentIndex) {
+        span.classList.add('current');
+      } else {
+        span.classList.remove('current');
+      }
+    });
+  }
+
```

Comment:
Instead of running another function that checks every span and updates the current, just use the current handleKeydown function. You are already checking your span at `currentIndex` and incrementing so we can do it in here to add it